testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/ManagementInterfaceResourcesTestCase.java

```diff
@@ -128,12 +146,31 @@ public void testTimeout() throws Exception {
     }
 
     private void runTest(int noRequestTimeout, ExceptionRunnable<Exception> test) throws Exception {
+        runTest(true, noRequestTimeout, test);
+        runTest(false, noRequestTimeout, test);
+    }
+
+    private void runTest(boolean useSystemProperties, int noRequestTimeout, ExceptionRunnable<Exception> test) throws Exception {
         controller.startInAdminMode();
         try (CLIWrapper cli = new CLIWrapper(true)) {
-            cli.sendLine(String.format("/system-property=%s:add(value=%d)", BACKLOG_PROPERTY, 2));
-            cli.sendLine(String.format("/system-property=%s:add(value=%d)", CONNECTION_HIGH_WATER_PROPERTY, 6));
-            cli.sendLine(String.format("/system-property=%s:add(value=%d)", CONNECTION_LOW_WATER_PROPERTY, 3));
-            cli.sendLine(String.format("/system-property=%s:add(value=%d)", NO_REQUEST_TIMEOUT_PROPERTY, noRequestTimeout));
+            if (useSystemProperties) {
+                cli.sendLine(String.format(SYSTEM_PROPERTY_ADD, BACKLOG_PROPERTY, 2));
+                cli.sendLine(String.format(SYSTEM_PROPERTY_ADD, CONNECTION_HIGH_WATER_PROPERTY, 6));
+                cli.sendLine(String.format(SYSTEM_PROPERTY_ADD, CONNECTION_LOW_WATER_PROPERTY, 3));
+                cli.sendLine(String.format(SYSTEM_PROPERTY_ADD, NO_REQUEST_TIMEOUT_PROPERTY, noRequestTimeout));
+            } else {
+                cli.sendLine(String.format(HTTP_INTERFACE_READ_ATTRIBUTE, BACKLOG_ATTRIBUTE), true);
```

Comment:
@yersan FYI at this point we do not need to reload the server for the test as our testsuites already run the server at the default stability level of the feature pack so here it is already running at the community level.

This is going to be a part of a bigger challenge, our WildFly and WildFly Core testsuites are going to need to remain runnable with the default configuration for the feature pack under test and for both these projects that will mean we will have community schemas in the configuration and we can't assume to run using the stability level of default.

To bridge this gap I have added this test to check if the attribute is present in the management model, if the attribute does not exist we assume we are running at a stability level that does not support it and skip that part of the test.

Reply:
> This is going to be a part of a bigger challenge, our WildFly and WildFly Core testsuites are going to need to remain runnable with the default configuration for the feature pack under test and for both these projects that will mean we will have community schemas in the configuration and we can't assume to run using the stability level of default.

My understanding about the initial approach to challenge this was that community schemas were not going to be present (they were merged but the plan is to rollback them) and any test which is not meant to run for "default" stability level should reload the server to the stability level they need. 

So, hence, if we provision to "community" stability level, the test will reload the server to the "community" stability level (if it is not already running on it) and the test will run. If we provision to "default" then the test will be skipped because the server will not support a reload to "community" stability level. This approach is what I had in mind that we were going to follow. If that is not correct or we require a different approach we should discuss what to do in Zulip.

> To bridge this gap I have added this test to check if the attribute is present in the management model

Ok, that's another approach we can use for this test.

Reply:
@yersan this test is unrelated to the issues with the existing schemas, this is a test that already exists in the manualmode testsuite and already tests using a server provisioned at the default stability level of the feature pack i.e. community.

Reply:
@darranl I think we are talking about different things. Now, with your latest changes skipping the test if the attribute is not present, the potential issue this PR could have introduced when we are converting to product branches is gone. That's fine. However, I was talking about what was the approach we planned to follow to face these cases: reload the server to the desired stability level if that stability level is not "default".

Reply:
> I have not added anything to this test to get the server to run at the Commnity stability level, in this module the server already runs at the default stability level of the feature pack there is nothing to need a reload to community as it is already at that stability level.

@darranl Without your change checking whether the attribute is present, we should have tweaked this test as soon as we provision the server to "default" stability level to create the product branches. However, the approach we defined to face this case was to reload the server to the desired stability level if that stability level is not "default". This PR is now approaching this problem differently, which is fine, but I only wanted to explain what we planned to do.

Reply:
@yersan I am still not following the place reloading to the community stability level would have a place on that workflow.

In this repository the server is provisioned using the community stability level and runs using the community stability level - reloading to the community stability level would be a noop as we end up where we started.

Once we are in conversion branches and beyond the default stability level doesn't just become the default stability level it becomes the only stability level so the reload to community again serves no purpose.

Reply:
@darranl Yes, let me try to explain this. If we use our current tasks to reload to a desired stability level, see https://issues.redhat.com/browse/WFCORE-6728:

1. The reload will be a noop if the target server is already running on the desired stability level; so in the end there won't be any overhead if we try to reload to "community" stability level a server that is already running at "community" level. Hence, in a nonproduct branch, the test will be executed as we were "not doing" a reload.
2. If the server does not support the desired stability level, for example, at product branches reload to "community" stability level won't be possible because the running server will only support "default" stability level; the test case will be ignored.

The final effect will be that the same code will make the test be ignored at product branches but be executed in nonproduct branches. Test configuration files should be kept using "default" stability level schemas. This "default" schemas can be read by a server that is running on "community" stability level or in "default" stability level.

Ignoring the test if the attributes don't exist is also a valid approach, in the end, what we should use will depend a lot on the test case itself. Having specific tests for community-only tests could make the code easier to promote to a higher stability level, since it promotion would be removing the task that reloads the server to the desired stability level, but I guess that should be done on more complex test cases. Its implementation is more complex ... and I don't know yet if it will cover all the cases or if we are missing something in that puzzle.

Reply:
@yersan thank you for the clarification, I understand where you are coming from.  I can move to that pattern in a follow up PR.

We maybe should add something to the table in the wildfly-proposals repo for the feature process describing the testing requirements (I need to check what we already have).

I think we should probably avoid going into the reload detail too much but I can see we should add some guidance discussing that all new tests should use the appropriate server setup task for the target stability level.

Reply:
> We maybe should add something to the table in the wildfly-proposals repo for the feature process describing the testing requirements (I need to check what we already have).

yes, that maybe could help